source/renderer/app/components/staking/widgets/ThumbPoolContent.js

```diff
@@ -44,22 +51,29 @@ export default class ThumbPoolContent extends Component<Props> {
     return (
       <div className={componentClassnames}>
         <div className={styles.ticker}>{ticker}</div>
-        {IS_RANKING_DATA_AVAILABLE ? (
-          <div className={styles.ranking} style={{ color }}>
-            {nonMyopicMemberRewards ? (
-              ranking
-            ) : (
-              <>
-                {numberOfRankedStakePools + 1}
-                <sup>*</sup>
-              </>
-            )}
-          </div>
-        ) : (
-          <div className={styles.noDataDash}>
-            <SVGInline svg={noDataDashBigImage} />
+        {isGridRewardsView && (
```

Comment:
What about using following format?
```{isGridRewardsView ? () : ()}```

Reply:
@yakovkaravelov then we will have nested ternary operators warning, linter won't allow that.

<img width="622" alt="Screen Shot 2021-03-04 at 4 28 56 AM" src="https://user-images.githubusercontent.com/15862062/109950534-3aeba500-7ca2-11eb-9556-c613847de72b.png">